### PR TITLE
Updated files to PEP8, documentation, format => f-string, refactor a couple of functions

### PR DESCRIPTION
**Pull request by FlymeDllVa**
- Updated files to PEP8, documentation
- Format => f-string
- Refactor functions
- Update shipping option

### CODE REVIEW

File: telebot/__init__.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 class Handler:
     """
-    Class for (next step|reply) handlers
+    Class for (next steplogger.debug|reply) handlers
```
Comment (Badiboy) [inline telebot/__init__.py]: Why "steplogger.debug"?

File: telebot/__init__.py [hunk 2]
```diff
@@ -141,31 +142,31 @@ def __init__(self, token, threaded=True, skip_pending=False, num_threads=2):
         """
 
         self.token = token
-        self.update_listener = []
+        self.update_listener = list()
```
Comment (Badiboy) [inline telebot/__init__.py]: What is the reason (here and below)? Nothing in PEP8 about it?
And here they say that it is slower: https://stackoverflow.com/questions/2972212/creating-an-empty-list-in-python

File: telebot/__init__.py [hunk 3]
```diff
@@ -311,7 +328,7 @@ def process_new_updates(self, updates):
             if update.pre_checkout_query:
                 new_pre_checkout_querys.append(update.pre_checkout_query)
 
-        logger.debug('Received {0} new updates'.format(len(updates)))
+        logger.debug("Received %s new updates", len(updates))
```
Comment (Badiboy) [inline telebot/__init__.py]: Change to f-string?

File: telebot/__init__.py [hunk 4]
```diff
@@ -449,15 +466,15 @@ def __non_threaded_polling(self, none_stop=False, interval=0, timeout=3):
                     self.__stop_polling.set()
                     logger.info("Exception occurred. Stopping.")
                 else:
-                    logger.info("Waiting for {0} seconds until retry".format(error_interval))
+                    logger.info("Waiting for %s seconds until retry", error_interval)
```
Comment (Badiboy) [inline telebot/__init__.py]: f-string?

File: telebot/__init__.py [hunk 5]
```diff
@@ -1314,12 +1594,17 @@ def _notify_next_handlers(self, new_messages):
...
 
-    def message_handler(self, commands=None, regexp=None, func=None, content_types=['text'], **kwargs):
+        :param handler:
+        :param filters:
+        :return:
+        """
+
+        return dict(function=handler, filters=filters)
+
+    def message_handler(self, commands=None, regexp=None, func=None, content_types=None, **kwargs):
```
Comment (Badiboy) [inline telebot/__init__.py]: Breaks backward compatibility.

File: telebot/__init__.py (hunk 4 above)
Comment (FlymeDllVa) [reply telebot/__init__.py]: https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs/54368086

File: telebot/__init__.py (hunk 2 above)
Comment (FlymeDllVa) [reply telebot/__init__.py]: Perhaps not really the best change. It just reads better.

File: telebot/__init__.py (hunk 1 above)
Comment (FlymeDllVa) [reply telebot/__init__.py]: Sorry I didn't see it

File: telebot/types.py [hunk 6]
```diff
@@ -445,44 +437,43 @@ def __init__(self, message_id, from_user, date, chat, content_type, options, jso
...
             return text
-        _subs = {
-            "bold"     : "<b>{text}</b>",
-            "italic"   : "<i>{text}</i>",
-            "pre"      : "<pre>{text}</pre>",
-            "code"     : "<code>{text}</code>",
-            "url"      : "<a href=\"{url}\">{text}</a>",
-            "text_link": "<a href=\"{url}\">{text}</a>"
-        }
+        _subs = dict(bold="<b>{text}</b>", italic="<i>{text}</i>", pre="<pre>{text}</pre>", code="<code>{text}</code>",
```
Comment (Badiboy) [inline telebot/types.py]: Single item per single line pls...

**Comment by Badiboy** [comment]
Итого. Просьба, если есть возможность:
1. Устранить согласованные вопросы
2. Разрешить конфликты с основной веткой

_PS. Благодаря уважаемому eternnoir у меня появились ключи от сейфа, можно попробовать освежить этот неплохой продукт. :)_

**Comment by Badiboy** [comment]
image

Problems with f-strings and python 2.7 & 3.5... 3.5 is not at EOL... :(

**Comment by FlymeDllVa** [comment]
:/

**Comment by Badiboy** [comment]
Hm. 16 more edited files instead of clearing previous ones. 

Me personally do not like the formatting style you bring. I did not like the previous one, I do not like new one even more. So I cannot accept this request, sorry. May be someone else did it.

I manually bring all (as far as I found them in 22 files) valuable updates to the pull request https://github.com/eternnoir/pyTelegramBotAPI/pull/728.

**Comment by FlymeDllVa** [comment]
Ok

**Comment by Badiboy** [comment]
No need to close the issue. I'm not the one with final decisions. It was my personal oppinion, owner may have another one.

**Pull request closed without merging**